.travis.yml

```diff
@@ -75,12 +75,12 @@ install: |
 
   docker exec -u 0:0 "${DOCKER_INSTANCE}" chown -R $(id -u $USER):$(id -g $USER) /home/jenkins
 
-script: |
-  docker exec -t -u $(id -u $USER):$(id -g $USER) "${DOCKER_INSTANCE}" make "${MAKE_TEST_TARGET}"
-
-  if [ "${WCT}" == "1" ]; then
-    cd webapp && npm test
-  fi
+script:
+  - docker exec -t -u $(id -u $USER):$(id -g $USER) "${DOCKER_INSTANCE}" make "${MAKE_TEST_TARGET}"
```

Comment:
Splitting these up exposed a bug that I introduced when I grouped them all together: since there is no `MAKE_TEST_TARGET` for the first entry of `matrix.include` array, this command doesn't need to be run—its failure was previously silenced by the command that follows. 

To fix this:

```js
- |
  if [ "${MAKE_TEST_TARGET}" != "" ]; then
    docker exec -t -u $(id -u $USER):$(id -g $USER) "${DOCKER_INSTANCE}" make "${MAKE_TEST_TARGET}"
  fi
```